app/ParserTests/ParserTests.hs

```diff
@@ -8,10 +8,26 @@ Module containing test cases for Requirement Parsers.
 module ParserTests.ParserTests
 ( reqTestSuite ) where
 
-import Test.HUnit
+import qualified Text.Parsec as Parsec
+import Text.Parsec.String (Parser)
+import Database.Requirement
+import WebParsing.ReqParser
+import Test.HUnit ( assertEqual, Test(..) )
 
-test1 :: Test
-test1 = TestCase (assertEqual "TEST PASS" ("True") ("True"))
+-- Function to facilitate test case creation given a string, Req tuple
+createTest :: (Eq a, Show a) => Parser a -> (String, a) -> Test
+createTest parser (input, expected) = let courseReq =  Parsec.parse parser "" input
+                                      in TestCase $ (assertEqual ("for (" ++ input ++ "),") (Right expected) courseReq)
+
+-- Basic inputs found on artsci website; only andParser.
+basicInputs :: [(String, Req)]
+basicInputs = [("CSC120H1/CSC148H1", OR [J "CSC120H1", J "CSC148H1"]),
+			   ("CSC120H1, CSC121H1, CSC148H1", AND [J "CSC120H1", J "CSC121H1", J "CSC148H1"]),
```

Comment:
I want you to format these tests properly and break them into multiple lines. Use the format

```haskell
basicInputs = [
    (...),
    (...),
    ...
    ]
```

Also don't be afraid to split individual test cases into multiple lines as well to ensure a reasonable line length.